editoast/src/core/infra_state.rs

```diff
@@ -16,7 +16,9 @@ pub struct InfraStateRequest {
 #[derive(Debug, Clone, Derivative, Deserialize, Serialize)]
 #[derivative(Default)]
 pub struct InfraStateResponse {
+    #[derivative(Default(value = r#""NOT_LOADED".into()"#))]
     pub last_status: String,
+    #[derivative(Default(value = r#""NOT_LOADED".into()"#))]
     pub status: String,
 }
```

Comment:
We shouldn't use `String` but `InfraState`. The `last_status` can be `null`.

```suggestion
#[derive(Debug, Clone, Copy, Default, Derivative, Deserialize, Serialize)]
pub struct InfraStateResponse {
    pub last_status: Option<InfraState>,
    pub status: InfraState,
}
```